.gitignore

```diff
@@ -8,3 +8,10 @@
 .Rproj.user
 *-Ex.R
 *.tif
+
+
+# Ignore data in modules
```

Comment:
to be fully modular, modules should use their own .gitignore files

Reply:
True, but since they haven't been converted to sub-module repos yet...

Reply:
No, they haven't but they should still be using their own .gitignore files :)

I want to switch things over to using submodules asap anyway